docs/scratch_orgs.rst

```diff
@@ -1,38 +1,253 @@
+
+Scratch org limits are based on your DevHub's edition and your Salesforce contract. To review limits and consumption:
+
+.. code-block:: console
+
+    $ sfdx force:limits:api:display -u <username>
+
+``<username>`` is your DevHub username. The limit names are ``ActiveScratchOrgs`` and ``DailyScratchOrgs``.
+
+
+
+
+List Orgs
 ---------
 
-Open an org in the browser
---------------------------
+When inside a project repository, run ``cci org list`` to see all the orgs you have configured or connected.
+
+
+
+Open Orgs in the Browser
+---------------------------
+
+Run ``cci org browser <org>`` to log into any org in the keychain in a new browser tab.
 
-Delete a scratch org
---------------------
 
-Set a default org
+
+Delete Scratch Orgs
+-------------------
+
+If an org defined in the keychain has created a scratch org, delete the scratch org but leave the configuration to regenerate it in the keychain.
+
+.. code-block:: console
+
+    $ cci org scratch_delete <org>
+
+Using ``scratch_delete`` doesn't remove the org ``<org>`` from your org list.  This default behavior lets you easily recreate scratch orgs from a stored, standardized configuration.
+
+To permanently remove an org from the org list:
+
+.. code-block:: console
+
+    $ cci org remove
+
+To delete a scratch org created from the config:
+
+.. code-block:: console
+
+    $ cci org remove <org>
+
+It's not necessary to explicitly remove or delete expired orgs. CumulusCI recreates an expired org the first time you attempt to use it. To clean up expired orgs from the keychain:
+
+.. code-block:: console
+
+    $ cci org prune
+
+
+
+Set a Default Org
 -----------------
 
-Add a predefined org
---------------------
+When you run a flow or task that performs work on an org, specify the org with an ``--org <org>`` flag.
+
+.. code-block:: console
+
+    $ cci flow run dev_org --org dev
+
+To run many commands against the same org, set a default.
+
+.. code-block:: console
+
+    $ cci org default dev
+    $ cci flow run dev_org
+
+Alternately, set a default org when creating a new named configuration.
+
+.. code-block:: console
+
+    $ cci org scratch dev <org> --default
+
+To remove an existing default:
 
-Import an org from the Salesforce CLI
+.. code-block:: console
+
+    $ cci org default dev --unset
+
+
+
+Configure Predefined Orgs
+-------------------------
+
+Projects can customize the set of configurations available out of the box, and add further predefined orgs to meet project-specific needs. 
+
+An org configuration has a name, such as ``dev`` or ``qa``, and is defined by options set in the ``cumulusci.yml`` file as well as the contents of a specific ``.json`` scratch org definition file in the ``orgs`` directory. For orgs like ``dev`` and ``qa`` that are predefined for all projects, the configuration is located in the CumulusCI standard library, but can be customized by projects in the ``cumulusci.yml`` file.
```

Comment:
"Specific ``.json`` scratch org definition file" -- Does this file have a name? Should it? Or should we list file path?

Reply:
[Scratch org definition file](https://developer.salesforce.com/docs/atlas.en-us.sfdx_dev.meta/sfdx_dev/sfdx_dev_scratch_orgs_def_file.htm#!) is the terminology that Salesforce uses. You can have multiple definition files, so we can't provide a specific name. We also indicate that these live in the `orgs` directory. I might add a `/` to that to more clearly indicate it's a directory in a filepath.
```suggestion
An org configuration has a name, such as ``dev`` or ``qa``, and is defined by options set in the ``cumulusci.yml`` file as well as the contents of a specific ``.json`` scratch org definition file in the ``orgs/`` directory. For orgs like ``dev`` and ``qa`` that are predefined for all projects, the configuration is located in the CumulusCI standard library, but can be customized by projects in the ``cumulusci.yml`` file.
```

Reply:
I was just confirming whether or not the specific .json file was easily pinpointed by way of a name or an filepath. No matter then.